### PR TITLE
Fix the STATICURL conundrum

### PR DESCRIPTION
**Pull request by palnabarun**

### CODE REVIEW

**Comment by palnabarun** [comment]
**NOTE: Anyone reading this, please ignore this PR**

**Comment by ananyo2012** [comment]
Looks good. @palnabarun Can you pull these changes to the server and see if it causes any issue with deployment.

File: pelicanconf.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 AUTHOR = u'PyCon India Content Team'
 SITENAME = u'In PyCon Blog'
-SITEURL = '/blog'
+SITEURL = '/'
```
Comment (ananyo2012) [inline pelicanconf.py:7]: You can simply leave this blank

Comment (ananyo2012) [reply pelicanconf.py:7]: Actually we don't need to update this at all. Using SITEURL in the links is the correct fix. The `/blog` url is needed in the devserver while running locally. In Netlify we can use the same `make publish` target. The `RELATIVE_URLS = True` setting overrides the SITEURL setting during publish and uses relative urls everywhere where SITEURL is used.

File: themes/inpycon2020/templates/base.html [hunk 2]
```diff
@@ -25,19 +25,19 @@
...
           </div>
         </div>
 
         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
           <ul class="nav navbar-nav main-menu">
             <li><a href="/2020/">Home</a></li>
-            <li class="current"><a href="/blog/">Blog</a></li>
-            <li><a href="/2020/coc.html">Code Of Conduct</a></li>
+            <li class="current"><a href="{{ SITEURL }}/">Blog</a></li>
+            <li><a href="https://in.pycon.org/2020/coc.html">Code Of Conduct</a></li>
```
Comment (palnabarun) [inline themes/inpycon2020/templates/base.html:40]: @vipulgupta2048 fyi, the COC URL. See the commit messages for the change.

File: pelicanconf.py (hunk 1 above)
Comment (palnabarun) [reply pelicanconf.py]: Resolved.

**Pull request merged**